hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -229,11 +228,15 @@
                  "+initialize_ram=1",
                  "+sw_test_timeout_ns=50000000"]
     }
-    // The test below is added in the included tl_access_tests.hjson.
-    // We just need to append the stub_cpu_mode run mode to it.
+    // The test below is from tl_access_tests.hjson, but we don't need to include the other tests
+    // like tl_intg_err. So add it here and append the stub_cpu_mode run mode.
```

Comment:
Do you think it is necessary to add some explanations as why we do not need to include the other tl_access tests?

Reply:
sure, updated the comment. Thanks